task.php

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Project Manager - Project Name</title>
+    <script src="https://cdn.tailwindcss.com"></script>
+</head>
+<body class="h-full">
+<header class="p-3 bg-teal-50 flex justify-between">
+    <h1 class="sm:text-5xl text-4xl"><a href="index.php">Project Manager</a></h1>
+    <div class="pr-3 flex">
+        <a href="task.php" class="p-3 bg-slate-300 rounded-l-lg border-y border-l">🇬🇧</a>
+    </div>
+</header>
+<main class="p-3">
+
+<?php
```

Comment:
You should put all the 'generic' stuff like this (everything in this php tag) at the very top of the file. 

It makes it quicker for you to see which other files/classes etc are being used